<launch>
    <!-- foot_switches -->
	<param name="foot_switches_publish_frequency_hz" value="20.0" />
    <node pkg="fetch" type="foot_switches" name="foot_switches" output="screen" />

    <!-- foot_motion --> 
    <node pkg="fetch" type="foot_servos" name="foot_servos" output="screen"/>
</launch>
